pontoon/messaging/models.py

```diff
@@ -53,3 +53,12 @@ class Meta:
 
     def is_new(self):
         return self.sent_at > timezone.now() - timezone.timedelta(minutes=1)
+
+
+class EmailContent(models.Model):
+    email = models.CharField(max_length=255, unique=True)
```

Comment:
Not sure about the naming here. Maybe that's better?

```suggestion
class EmailTemplate(models.Model):
    type = models.CharField(max_length=255, unique=True)
```